pom.xml

```diff
@@ -1612,6 +1612,11 @@
             </goals>
           </execution>
         </executions>
+        <configuration>
+          <includes>
+            <include>org/apache/pinot/**/*</include>
```

Comment:
Should we only include java files?

Reply:
per jacoco documentation it should be fine to assume jacoco only annotates code.
that being said we might need to deal with the problem of generated code going forward. but I would leave that to future if it becomes an issue